tslint.json

```diff
@@ -0,0 +1,39 @@
+{
+  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier", "tslint-immutable"],
+  "rules": {
+    "interface-name": false,
```

Comment:
This should be either `[true, "always-prefix"]` or `[true, "never-prefix"]`, right?

As I think about it a bit more I think it would be cool to have it prefixed because you would see that this is the interface instantly.

Reply:
Hey, I would NOT prefix the interface name with `I`, as this is not part of the TypeScript style guide -- which I think is the autoritive source here. We should do everything to ensure the code is adhering to TypeScript ecosystem best practices.